.ci/travis/travis.sh

```diff
@@ -105,12 +97,40 @@ javac8)
   done
   ;;
 
-jdk12-assembly-site)
+javac9)
+  files=($(grep -Rl --include='*.java' ': Compilable with Java9' \
+        src/test/resources-noncompilable || true))
+  if [[  ${#files[@]} -eq 0 ]]; then
+    echo "No Java9 files to process"
+  else
+      mkdir -p target
+      for file in "${files[@]}"
+      do
+        javac -d target "${file}"
+      done
+  fi
+  ;;
+
+javac13)
+  files=($(grep -Rl --include='*.java' ': Compilable with Java13' \
+        src/test/resources-noncompilable || true))
+  if [[  ${#files[@]} -eq 0 ]]; then
+    echo "No Java13 files to process"
+  else
+      mkdir -p target
+      for file in "${files[@]}"
+      do
+        javac --release 13 --enable-preview -d target "${file}"
```

Comment:
Do we need `--release 13`  explicitly ?
I hope eventually compilation will become the same for all jdk with only difference in version number, defined one's in code block or just taken from `java -version`

Reply:
We need `--enable-preview` for text blocks, which requires `--release` or `--source` as described here:
https://docs.oracle.com/en/java/javase/13/docs/specs/man/javac.html

Reply:
Thanks for explaining